### PR TITLE
[TASK] Create full command-json list using dependency

### PR DESCRIPTION
**Pull request by garvinhicking**
This needs to install a stub TYPO3 installation (SQLite) so that all CLI commands can be passed through a working TYPO3 environment. This is achieved with the "make setup-typo3" command.

### CODE REVIEW

**Comment by linawolf** [comment]
Ill have a look later. Thanks a lot for working on this!!

File: composer.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "license": "OPL-1.0",
     "type": "typo3-cms-documentation",
     "require": {
+        "garvinhicking/clinspector-gadget": "^0.1.1",
```
Comment (linawolf) [inline composer.json:7]: Is there a specific reason you added this to require, not to require-dev?

Comment (linawolf) [reply composer.json]: I added a commit to move it. drop my commit if there is a specific reason

**Comment by linawolf** [comment]
I added two commits:

One displays all the commands instead of just the list command so that the result of this PR can already be viewed

One moves the garvinhicking/clinspector-gadget to the dev-dependencies in the composer.json as it is only used in dev-context

**Review by linawolf — DISMISSED** [review]

**Comment by garvinhicking** [comment]
Cool, thank you. The missing 'require-dev' was no specific intention. Feel free to merge!

**Comment by linawolf** [comment]
I would like to exclude the example and styleguide commands before merging if that is ok for you

**Comment by linawolf** [comment]
currently working on improving the render-guides for that

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
@garvinhicking did I understand that correctly that you will make the backports manually?

**Pull request merged**